locales-pending/settings-premium.ftl

```diff
@@ -7,3 +7,14 @@
 settings-cancel-plus-title = Cancel { -brand-monitor-plus } subscription
 settings-cancel-plus-details = Your subscription will revert to a free account after the current billing cycle ends. Your data broker scan results will be permanently deleted, and you may be re-added to those sites.
 settings-cancel-plus-link-label = Cancel from your { -brand-mozilla-account }
+
+## Delete Monitor account
+
+settings-delete-monitor-plus-account-title = Delete { -brand-monitor } account
+settings-delete-monitor-plus-account-description = This will permanently delete your { -brand-monitor } account and immediately end your paid { -brand-monitor-plus } subscription.
+settings-delete-monitor-plus-account-cta-label = Delete account
+settings-delete-monitor-plus-account-dialog-title-2 = Your { -brand-monitor } account will be deleted
+settings-delete-monitor-plus-account-dialog-lead-p1-2 = All of your { -brand-monitor } account information will be deleted and we’ll no longer monitor for new data breaches or data broker exposures. This will not delete your { -brand-mozilla } account.
```

Comment:
```suggestion
settings-delete-monitor-plus-account-dialog-lead-p1-2 = All of your { -brand-monitor } account information will be deleted and we’ll no longer monitor for new data breaches or data broker exposures. This will not delete your { -brand-mozilla-account }.
```

Reply:
Aargh, not again. Fixed in 700e7e2b4e7796c2d75aa2bfa79183ede0341f1b.

I also landed two new strings after discussion with content in 23d3b3a114c63e04386697185786a4a3a1208d7d.

Reply:
Possibly [8&times;] other legacy strings in en* or pending strings:
```sh
git grep -in "{ -brand-mozilla } account" -- "locales-pending/*.ftl" "locales/en*/*.ftl" | cat

locales/en-GB/settings.ftl:66:settings-delete-monitor-free-account-dialog-lead = All of your { -brand-monitor } account information will be deleted and we’ll no longer monitor for new data breaches. This will not delete your { -brand-mozilla } account.
locales/en-GB/settings.ftl:73:settings-delete-monitor-plus-account-dialog-lead-p1 = All of your { -brand-monitor } account information will be deleted and we’ll no longer monitor for new data breaches or data broker exposures. This will not delete your { -brand-mozilla } account.
locales/en/settings.ftl:68:settings-delete-monitor-free-account-dialog-lead = All of your { -brand-monitor } account information will be deleted and we’ll no longer monitor for new data breaches. This will not delete your { -brand-mozilla } account.
locales/en/settings.ftl:75:settings-delete-monitor-plus-account-dialog-lead-p1 = All of your { -brand-monitor } account information will be deleted and we’ll no longer monitor for new data breaches or data broker exposures. This will not delete your { -brand-mozilla } account.
```

Reply:
No point in changing existing strings just to fix the brand name, we need to fix them as we update them for other reasons.